templates/collectd.conf.j2

```diff
@@ -2,7 +2,8 @@
 FQDNLookup true
 
 AutoLoadPlugin false
-TypesDB "/usr/share/collectd/types.db"
+
+TypesDB "/usr/share/collectd/types.db"{% if "libpodstats" in collectd_plugins %} "/usr/share/collectd/types.db.libpodstats"{% endif %}
```

Comment:
Is there any reason to make this conditional? I feel like this line should includes the types dbs for all supported plugins by default unless there is an obvious downside.

Reply:
The only downside I can think of is if the libpodstats plugin is not installed, collectd will fail because it cannot find the file. That's the case now, but should not be when we actually use ansible instead of heat templates.  

Another _could_ arise if a different plugin tried to define the `pod_cpu` and `pod_memory` types, but that is unlikely to happen.

Reply:
Double-check that this doesn't destroy the newline at the end of this line (putting "Interval" up on the same line). I think that's the problem that's breaking my HA STO PR.

Reply:
@pleimer Have you had a change to test this yet? Please see https://github.com/infrawatch/service-telemetry-operator/pull/122#pullrequestreview-485424919  and LMK if you find another solution.

Reply:
I just ran a test and it has the same issue you mentioned in the other PR

Reply:
I added an extra newline after it and it formats it properly